src/pycolmap/helpers.h

```diff
@@ -406,3 +407,28 @@ inline bool IsPyceresAvailable() {
   }
   return true;
 }
+
+template <typename Parent>
+inline void DefDeprecation(
+    Parent& parent,
+    std::string old_name,
+    std::string new_name,
+    std::optional<std::string> custom_warning = std::nullopt) {
+  parent.def(
+      old_name.c_str(),
+      [parent,
+       old_name = std::move(old_name),
+       new_name = std::move(new_name),
+       custom_warning = std::move(custom_warning)](const py::args& args,
+                                                   const py::kwargs& kwargs) {
+        if (custom_warning) {
+          PyErr_WarnEx(PyExc_DeprecationWarning, custom_warning->c_str(), 1);
```

Comment:
Does this print every single time it is called? if so, should we use `LOG_FIRST_N(WARNING, N)` instead?

Reply:
No, Python's deprecation warnings only get issued once upon first usage, so it is equivalent to LOG_FIRST_N() with the added benefit of working nicely together with Python's warning filtering system, etc.